workflow/scripts/tif_stacks_to_ome_zarr.py

```diff
@@ -62,18 +64,46 @@
 darr_channels = da.stack(darr_list)
 
 
-store = zarr.DirectoryStore(out_zarr)
-root = zarr.group(store,path='/',overwrite=True)
-scaler = Scaler(max_layer=max_layer,method=scaling_method)
+if snakemake.config['write_to_remote']:
+    #use the uri
+    uri = snakemake.params.uri
+
+    if uri.startswith('gcs://'):
+        uri = uri[6:]
```

Comment:
looking back at this, there is probably a more elegant way of doing this, I think I did it in a hurry..